pages/index.tsx

```diff
@@ -31,10 +34,28 @@ function Homepage() {
     setTimeEntries(response);
   };
 
+  const fetchClients = async () => {
+    const response = await getClients();
+    if (response instanceof NotFoundError) {
+      setIsDataError(true);
+      return;
+    }
+    setClients(response);
```

Comment:
I see you made a new fetch function to get all the clients from the DB. 
But, what happens if you want to add a client?

Reply:
Discussed already, is resolved.